label-inspector/data/output/batch_inference/sample_batch_dataset_1.csv.out

```diff
@@ -0,0 +1,201 @@
+letter_grade,A_probability,B_probability,C_probability,D_probability,F_probability
```

Comment:
what are your thoughts on renaming this column to be "prediction" instead of the label-column name?

Did more of the offerings you saw on Marketplace return predictions with the header being same name as the label column or some fixed name like "prediction" ?

Reply:
Did not have a strong preference either, but changed to `predicted_label` bc I believe studio was doing something similar (I might be wrong, but either way I think it is understandable